packages/fx-core/src/common/localizeUtils.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Locale } from "../core/globalVars";
+import { getResourceFolder } from "../folder";
+import * as path from "path";
+import fs from "fs-extra";
+import * as util from "util";
+
+export function getLocalizedString(key: string, ...params: any[]): string {
```

Comment:
Maybe allow a default value if no key defined in package.nls.json

Reply:
We can just return empty string or throw an exception if the key does not exist. I don't think it is meaningful that developer provide a default value, if the value does not exist, he should add it in the package.nls.json file

Reply:
Yes, that's a good way to make sure developer added the string in package.nls.json. My concern is the UI might get broken if something wrong with the json file.

Reply:
I think UI will not broken with an empty message.

Reply:
Please make sure all strings are extracted to json file when reviewing PR.